src/mockServer/fixtures/diseaseDTO.js

```diff
@@ -0,0 +1,13 @@
+const disease1DTO = {
```

Comment:
Maybe there's a typo in this identifier?

Reply:
I replaced that to `
 export const diseaseDTO = {....}
`
but the intention there was to create several objects, desease1, desease2, desease3 for mock purpose